WooCommerce/WooCommerceTests/ViewModels/CardPresentPayments/CollectOrderPaymentUseCaseTests.swift

```diff
@@ -77,7 +77,7 @@ final class CollectOrderPaymentUseCaseTests: XCTestCase {
         assertEqual(.foundReader, mockAnalyticsTracker.spyPaymentCancelationSource)
     }
 
-    func test_collectPayment_processing_completion_tracks_payment_success_event() async throws {
+    func test_collectPayment_processing_completion_tracks_payment_success_event() throws {
```

Comment:
Do you know why would the test be slower just because (miss)uses the `async` keyword? There's no await inside so it shouldn't be any different than a non-async function, no?

Reply:
I'm not sure, there may be no difference. There's some weirdness and inconsistency around these tests and the results differ between my local machine and CI. Either way, since `async` is not used we can remove it.